Cyotek.Windows.Forms.ImageBox/ImageBox.cs

```diff
@@ -1389,6 +1403,28 @@ public virtual bool LimitSelectionToImage
       }
     }
 
+    /// <summary>
+    /// Gets or sets the how the mouse wheel is handled
+    /// </summary>
+    /// <value>
+    /// The pan mode.
+    /// </value>
+    [Category("Behavior")]
+    [DefaultValue(typeof(ImageBoxMouseWheelMode), "Both")]
```

Comment:
"Both" isn't valid for the enum you've added. In addition, it would be a breaking change to have the default as `ScrollAndZoom`, so for this particular version at least the default must remain at `Zoom` and users must opt in to this new behaviour.

Reply:
Damn, thats a good catch. This is just a copy and paste issue. I am going to set "Zoom" here to reatin compatibility.

Reply:
I caught that, but I did miss the documentation error which still refers to pan mode :) No big deal, I'll tweak that when I do the version updates etc.

Reply:
Fixed :-)